### DISCUSSION TITLE
SafeExamBrowser Start with Screen over

### DISCUSSION THREAD

**Discussion by venom-marvel** [SafeExamBrowser/seb-win-refactoring#173] [Q&A]
Hello all.
For me, the new SEB browser version always starts with this welcome screen. how do I get rid of it. It is mega annoying and not what I expect from a kiosk mode.

Maybe someone has a tip

Here is the Picture:
https://ibb.co/P1X71bs

**Comment by danschlet** [comment]
Almost all applications I know display some kind of splash screen (window) with the name and version of the application. What exactly is your problem with this being displayed?

**Reply by venom-marvel** [reply]
It will stay and need to close by finger. So if my kiosk autostart, it will hide everything with seb splash.

**Reply by danschlet** [reply]
Ok, that should not happen. Maybe @dbuechel can investigate the issue.

**Reply by dbuechel** [reply]
The only reason this would happen is if you run SEB without a kiosk mode (in the configuration tool, check the kiosk mode you're using under tab "Security").

**Reply by venom-marvel** [reply]
Yes! This was the problem. Thank you very much. Thanks to all.
